pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -576,6 +576,60 @@ func NewVirtualMachineClusterFlavorCrd() (*extv1.CustomResourceDefinition, error
 	return crd, nil
 }
 
+func NewVirtualMachinePreferenceCrd() (*extv1.CustomResourceDefinition, error) {
+	crd := newBlankCrd()
+
+	crd.Name = "virtualmachinepreferences." + flavorv1alpha1.SchemeGroupVersion.Group
+	crd.Spec = extv1.CustomResourceDefinitionSpec{
+		Group: flavorv1alpha1.SchemeGroupVersion.Group,
+		Names: extv1.CustomResourceDefinitionNames{
+			Plural:     flavor.PluralPreferenceResourceName,
+			Singular:   flavor.SingularPreferenceResourceName,
+			ShortNames: []string{"vmp", "vmps"},
```

Comment:
nit: would change to `vmpref` or something a bit more specific.

Reply:
ACK, I'll add both.